_topic_map.yml

```diff
@@ -310,19 +310,21 @@ Topics:
 ---
 Name: Networking
 Dir: networking
-Distros: openshift-enterprise,openshift-origin
+Distros: openshift-enterprise,openshift-origin,openshift-dedicated
 Topics:
 - Name: Understanding networking
   File: understanding-networking
 - Name: Understanding the Cluster Network Operator (CNO)
   File: cluster-network-operator
+  Distros: openshift-enterprise,openshift-origin
 - Name: Understanding the DNS Operator
   File: dns-operator
+  Distros: openshift-enterprise,openshift-origin
 - Name: Understanding the Ingress Operator
   File: ingress-operator
+  Distros: openshift-enterprise,openshift-origin
 - Name: Configuring network policy
   File: configuring-networkpolicy
-  Distros: openshift-origin,openshift-enterprise
 - Name: Multiple networks
```

Comment:
Is this one valid for dedicated?

Reply:
Yes, that is one we are adding in for dedicated.